node/trieIterators/stakeValuesProcessor_test.go

```diff
@@ -136,30 +120,14 @@ func TestTotalStakedValueProcessor_GetTotalStakedValue_CannotGetAccount(t *testi
 	require.Equal(t, expectedErr, err)
 }
 
-func TestTotalStakedValueProcessor_GetTotalStakedValueNilRootHashShouldError(t *testing.T) {
-	t.Parallel()
-
-	arg := createMockArgs()
-	arg.BlockChain = &testscommon.ChainHandlerStub{
-		GetCurrentBlockRootHashCalled: func() []byte {
-			return nil
-		},
-	}
-	totalStakedProc, _ := NewTotalStakedValueProcessor(arg)
-
-	resTotalStaked, err := totalStakedProc.GetTotalStakedValue()
-	require.Nil(t, resTotalStaked)
-	require.Equal(t, ErrNodeNotInitialized, err)
-}
-
 func TestTotalStakedValueProcessor_GetTotalStakedValue_CannotRecreateTree(t *testing.T) {
 	t.Parallel()
 
 	expectedErr := errors.New("expected error")
 	arg := createMockArgs()
 	arg.Accounts.AccountsAdapter = &stateMock.AccountsStub{
-		RecreateTrieCalled: func(rootHash []byte) error {
-			return expectedErr
+		GetExistingAccountCalled: func(addressContainer []byte) (vmcommon.AccountHandler, error) {
```

Comment:
might rename the suffix of this test's name to `CannotGetExistingAccount`

Reply:
already did. Thanks

Reply:
Found after self-reviewing this